### PR TITLE
Fix version.h problems for Linux

### PR DESCRIPTION
**Pull request by atellyro**
Signed-off-by: Harald <fechsaer@gmail.com>

### CODE REVIEW

**Comment by behrisch** [comment]
Thank you for the PR! Is this only to fix things when you have an installed and a developer version?

File: tools/sumolib/version.py [hunk 1]
```diff
@@ -25,15 +25,22 @@
...
 
 
 def _findVersion():
+    if verbose:
+      print('gitdir1 = %s' % GITDIR)
+      print('pwd1 = %s'    % os.getcwd())
     # try to find the version in the config.h
     versionFile = join(dirname(__file__), '..', '..', 'include', 'version.h')
+    if not exists(versionFile):
+      versionFile = join('src', 'version.h')
```
Comment (behrisch) [inline tools/sumolib/version.py]: There should be no version.h in src anymore. This is an artifact created only by "in tree" builds which are either using autotools or a discouraged cmake configuration.

**Comment by atellyro** [comment]
This script runs with pwd = .../build/cmake-build
So it picks up the version file generated in the build dir

Am 18.12.19 um 09:26 schrieb Michael Behrisch:

**Comment by atellyro** [comment]
Am 18.12.19 um 09:23 schrieb Michael Behrisch:
>
> Thank you for the PR! Is this only to fix things when you have an 
> installed and a developer version?
>
Yes,  except:

   you do some structural changes to sumolib.version.  In this case the 
eggs are called first, independent from PYTHONPATH etc.

   in this case you have to remove the installed sumolib

**Comment by behrisch** [comment]
thanks for the PR. I tried to solve the problems without adding too mucg debugging code. Please review whether your expectations are met.

**Pull request closed without merging**